packages/api/src/User/User.service.ts

```diff
@@ -36,4 +38,74 @@ export class UserService {
     user.profileImage = `https://www.gravatar.com/avatar/${md5(input.email)}`;
     return this.userRepository.save(user);
   }
+
+
+  /**
+   * Did the user complete a module yesterday (normalize to 12am)
+   * Returns:
+   *  true = Skipped
+   *  false = Not skippped. Increment
+   *  null = Not skipped. Already completed today. Don't increment
+   * @param userId User to check
+   */
+  private async _skippedDay(userId: string): Promise<boolean | null> {
```

Comment:
- Function name would be more descriptive as `_moduleCompletedYesterday`.
- Returning `null` is confusing, either they completed a module yesterday or they didn't; can you make this return either `true` or `false` only?